services/src/floodlight-modules/src/main/java/org/openkilda/floodlight/kafka/RecordHandler.java

```diff
@@ -162,11 +164,37 @@ private void handleCommand(CommandMessage message, String replyToTopic, Destinat
             doFlowVerificationRequest(context, (UniFlowVerificationRequest) data);
         } else if (data instanceof DeleteMeterRequest) {
             doDeleteMeter(message, replyToTopic, replyDestination);
+        } else if (data instanceof PortConfigurationRequest) {
+            doConfigurePort(message, replyToTopic, replyDestination);
         } else {
             logger.error("unknown data type: {}", data.toString());
         }
     }
 
+    private void doConfigurePort(final CommandMessage message, final String replyToTopic, 
+            final Destination replyDestination) throws SwitchOperationException {
+        PortConfigurationRequest request = (PortConfigurationRequest) message.getData();
+        
+        try {
+            logger.info("Port configuration request. Switch '{}', Port '{}'", request.getSwitchId(), 
+                    request.getPortNo());
+            ISwitchManager switchManager = context.getSwitchManager();
+            PortConfigurationResponse response = switchManager.configurePort(request);
+
+            InfoMessage infoMessage = new InfoMessage(response, message.getTimestamp(),
+                    message.getCorrelationId());
+            context.getKafkaProducer().postMessage(replyToTopic, infoMessage);
+        } catch (Exception e) {
+            logger.info("Port configuration request. Switch '{}', Port '{}'", request.getSwitchId(), 
```

Comment:
Why do you use info lever for errors? Also the message doesn't say that something went wrong.

Reply:
Looks like just copy/paste from the line 179

Reply:
Corrected. Exception Code was copied from doDeleteMeter method. Please update logging level there as well.